server/requirements.txt

```diff
@@ -1,5 +1,5 @@
 astroid==2.4.2
-astropy==4.2
+astropy==4.1
```

Comment:
why?

Reply:
![image](https://user-images.githubusercontent.com/25331712/111693786-53090b80-87ee-11eb-8ca7-35309da9c157.png)

Mentioned this in the PR. Not sure what the solution is.

Reply:
Ok. We can just use the old one